mp4parse/src/lib.rs

```diff
@@ -1123,11 +1650,12 @@ enum Extent {
     ToEnd { offset: u64 },
 }
 
+#[repr(C)]
 #[derive(Debug, PartialEq)]
 pub enum TrackType {
-    Audio,
-    Video,
-    Metadata,
+    Video = 0,
+    Audio = 1,
+    Metadata = 2,
```

Comment:
To respond to [your comment](https://github.com/mozilla/mp4parse-rust/pull/297#issuecomment-818031114):
> Tests are failing, because the `Default` for `TrackType` changed from `Video` to `Unknown`, I think the correct default is `Unknown`. Should I update the test in `minimal_mp4_get_track_info_invalid_track_number` to expect `Unknown` track type? This seems logical, but it changes the behavior of the C api, so just wanted to make sure.

I don't think we should collapse `Mp4parseTrackType` into `TrackType`. The two are distinct, and I think that's intentional, because `TrackType` has the possibility of being the `Unknown` variant, whereas `Mp4parseTrackType` does not. Unless there's a valuable reason to expose that to the C API, I'd say it's better not to.

Do you have a use case which requires it?